### ISSUE TITLE
Personal.fbs issue

### ISSUE THREAD

**Issue by MasterDriverRex** [pkZukan/PokeDocs#2]
The personal.fbs schema is useful, but it doesn't expand the 'evolutions' part into anything useful - it just counts the entire thing as a uint32 block labelled as "evo_table", making it almost impossible to alter evolution methods.
Meanwhile, the pknx dump is far more helpful, in separating it into different values - "Level", "Method", "Argument", etc.

Could this please be added to the personal.fbs file stored here?

**Comment by Reisyukaku** [comment]
Thanks for pointing that out, everything should be fixed 4e579e4151db2f7391206d330a42105642bf3f26

**Comment by MasterDriverRex** [comment]
Thanks for the update, but it's not working as expected - if you go to "species_index": 954 (Pawmi, for example), this is what is generated by your code:

```
"evo_table": {
        "level": 252,
        "method": 0,
        "arg": 96,
        "res0": 0,
        "res1": 80,
        "res2": 0,
        "species": 72,
        "form": 0 
}
```

This is incorrect, as there is no Lv. 252 and Pawmi evolves at Lv. 18.
The pknx version, listed under ""DexIndexNational": 954", has the following:

```
"Evolutions": [
        {
          "Level": 18,
          "Method": 4,
          "Argument": 0,
          "Reserved3": 0,
          "Reserved4": 0,
          "Reserved5": 0,
          "Species": 955,
          "Form": 0 
}
```
Needless to say, this is far more correct.
